src/kubernetes/grafana/grafana-deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: grafana
+  name: grafana
+spec:
+  selector:
+    matchLabels:
+      app: grafana
+  template:
+    metadata:
+      labels:
+        app: grafana
+    spec:
+      containers:
+        - image: springcloud/spring-cloud-dataflow-grafana-prometheus:2.0.0.BUILD-SNAPSHOT
```

Comment:
I hope we are targeting a release version for this before GA.

Reply:
I think this will be part of https://github.com/spring-cloud/spring-cloud-dataflow/pull/2904

Reply:
right; more specifically an item in issue https://github.com/spring-cloud/spring-cloud-dataflow/issues/2900 rather than the PR image building / publishing needs to be built into CI in one way or another. 

might need to be handled as we do elsewhere, i.e., the following get updated:

* https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/src/kubernetes/server/server-deployment.yaml#L19

* https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/src/kubernetes/skipper/skipper-deployment.yaml#L19